src/config/logger.cjs

```diff
@@ -0,0 +1,94 @@
+const winston = require('winston');
+const { format, transports, createLogger } = winston
+
+const DailyRotateFile = require('winston-daily-rotate-file');
+
+const fs = require('fs');
+const os = require('os');
+const homeDir = os.homedir();
+const logDir = `${homeDir}/.chia/climate-warehouse/logs`;
+
+if (!fs.existsSync(logDir)) {
+  fs.mkdirSync(logDir);
```

Comment:
hey this might be as simple as this: `fs.mkdirSync(targetDir, { recursive: true });`

Reply:
to fix the tests